channels/by.m3u

```diff
@@ -1,27 +1,35 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="8kanal.by" tvg-country="BY" tvg-language="" tvg-logo="" group-title="",8 канал (Витебск) (576p) [Not 24/7]
+#EXTINF:-1 tvg-id="8kanal.by" tvg-country="BY" tvg-language="Russian" tvg-logo="https://i.imgur.com/VpWmrEw.png" group-title="Local",8 канал (Витебск) (576p) [Not 24/7]
 http://95.46.208.8:24433/art
-#EXTINF:-1 tvg-id="FirstMusicChannelBielorrusia.by" tvg-country="BY" tvg-language="" tvg-logo="" group-title="Music",First Music Channel Bielorrusia (576p)
-http://hz1.teleport.cc/HLS/SD.m3u8
-#EXTINF:-1 tvg-id="Oneby.by" tvg-country="BY" tvg-language="" tvg-logo="" group-title="",One.by (2160p)
-http://rtmp.one.by:1499
```

Comment:
![image](https://user-images.githubusercontent.com/5430116/135622317-d6a852a4-1c2c-4a72-9d3f-aded6d4c46e2.png)
Very laggy but makes sense given that's a 4k channel